### PR TITLE
Validator and Assimilator Files

### PR DESCRIPTION
**Pull request by Charles-Peeke**

### CODE REVIEW

**Comment by Charles-Peeke** [comment]
The extra commits contain small changes and edits used to test/make the assimilator and validator

File: src/regulations_assimilator.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+        Simple checker to see if the file needs to be created or appended to.
+        """
+        if os.path.exists(filePath): 
+                append_write = 'a'
+        else: 
+                append_write = 'w'
+        return append_write
+
+
+# TODO
```
Comment (dstocker) [inline src/regulations_assimilator.py]: Is there anything else TODO here?

File: src/regulations_assimilator.py [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+server_key = os.environ["WORK_SERVER_KEY"]
+
+"""
+This program does the assimilation for Boinc. 
+It creates a master file of squares and adds the results of all of the work to the file.
+:return:
+"""
+
+
+def checkFile(filePath):
```
Comment (dstocker) [inline src/regulations_assimilator.py]: Might want to add documentation to the methods

File: src/regulations_validator.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+import sys, os, os.path, tempfile
+
+"""
+This program is a validator
+"""
+
+
+def file_length_checker(file_name):
```
Comment (dstocker) [inline src/regulations_validator.py:9]: Documentation for the methods would also be helpful here

**Pull request merged**